packages/components/src/internal/components/search/FindDerivativesButton.tsx

```diff
@@ -0,0 +1,156 @@
+import React, {FC, memo, useCallback, useMemo} from 'react';
+
+import { Filter } from '@labkey/api';
+
+import { QueryModel } from '../../../public/QueryModel/QueryModel';
+
+import { AppURL } from '../../url/AppURL';
+
+import { FIND_SAMPLES_BY_FILTER_KEY } from '../../app/constants';
+
+import { formatDateTime } from '../../util/Date';
+
+import { EntityDataType } from '../entities/models';
+
+import { ResponsiveMenuButton } from '../buttons/ResponsiveMenuButton';
+
+import { DataClassDataType, SampleTypeDataType } from '../entities/constants';
+
+import { SCHEMAS } from '../../schemas';
+
+import { getSampleFinderLocalStorageKey, isValidFilterFieldSampleFinder, searchFiltersToJson } from './utils';
+import {FieldFilter, FilterProps} from './models';
+import { SAMPLE_FINDER_SESSION_PREFIX } from './constants';
+import { useAppContext } from '../../AppContext';
+import {DisableableMenuItem} from "../samples/DisableableMenuItem";
+import {DisableableButton} from "../buttons/DisableableButton";
+
+const DISABLED_FIND_DERIVATIVES_MSG = 'Unable to find derivative samples using search filters or filters on multi-valued lookup fields';
+
+export const getFieldFilter = (model: QueryModel, filter: Filter.IFilter): FieldFilter => {
+    const colName = filter.getColumnName();
+    const column = model.getColumn(colName);
+
+    return {
+        fieldKey: column?.fieldKey ?? colName,
+        fieldCaption: column?.caption ?? colName,
+        filter,
+        jsonType: column.isLookup() ? 'string' : column?.jsonType ?? 'string', // deferring to 'string' for lookups since lookup display columns default to text fields
```

Comment:
should we use column.jsonType or column.displayFieldJsonType here? Sample Finder uses displayFieldJsonType

Reply:
Perfect, yes I'll switch to `column.displayFieldJsonType`. I didn't see that in the props before. Thanks.